api/autopilot.go

```diff
@@ -34,11 +35,26 @@ type (
 	}
 
 	HostHandlerGET struct {
-		ScoreBreakdown  HostScoreBreakdown `json:"scoreBreakdown"`
-		Score           float64            `json:"score"`
-		Usable          bool               `json:"usable"`
-		UnusableReasons []string           `json:"unusableReasons"`
-		Host            hostdb.Host        `json:"host"`
+		Host hostdb.Host `json:"host"`
+
+		Gouging          bool                 `json:"gouging"`
+		GougingBreakdown HostGougingBreakdown `json:"gougingBreakdown"`
+		Score            float64              `json:"score"`
+		ScoreBreakdown   HostScoreBreakdown   `json:"scoreBreakdown"`
+		Usable           bool                 `json:"usable"`
```

Comment:
I considered flipping this to `Unusable` and making it `UnusableBreakdown` to make it consistent with the rest. Not sure if it's worth it

Reply:
It's probably fine to keep it as is.